src/ontology/mondo-ingest.Makefile

```diff
@@ -553,12 +553,11 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 ##################################
 ## Externally managed content ####
 ##################################
+# General
+update-externally-managed-content: external-content-nord external-content-nando external-content-ordo external-content-omim
```

Comment:
## Why `external/`?
I still don't understand why ORDO and OMIM are in "externally managed content" and follow this pattern. They are not wholly externally managed. I guess I still haven't been given the full rundown of this pipeline and how it is different than the other parts of how `mondo` and `mondo-ingest` interact with our sources; I just have a very general idea.

Reply:
Externally managed content means that

1. The content is provided by an external resource
2. We dont review it - we just incorporate it

Why do to you think the are not "wholly externally managed"? Because we extract the information and transform it a bit?

Reply:
> Why do to you think the are not "wholly externally managed"? Because we extract the information and transform it a bit?

Yes, partly. I guess I wasn't entirely sure that this stuff wasn't going to be reviewed. Normally when I make a robot template, the outputs undergo some scrutiny. But I see now that this is the intent of everything we're putting in `external/`. We are trusting it / not reviewing it. Makes sense to me now.